.github/workflows/test.yml

```diff
@@ -144,7 +147,7 @@ jobs:
 
         run: cabal test wrapper-test --test-options="$TEST_OPTS --rerun-log-file .tasty-rerun-log-wrapper" || cabal test wrapper-test --test-options="$TEST_OPTS  --rerun-log-file .tasty-rerun-log-wrapper" || cabal test wrapper-test --test-options="$TEST_OPTS  --rerun-log-file .tasty-rerun-log-wrapper"
 
-      - if: matrix.test && matrix.ghc != '9.0.1'
+      - if: matrix.test && matrix.ghc != '9.0.1' && matrix.ghc != '9.0.2'
```

Comment:
I wonder what would happen if we did this unconditionally but set the test suite to not be buildable on GHC 9 :thinking: might error, might just succeed...

Reply:
hmm i think it is good to build it even if the tests are gonna fail, or it will rot even with compile errors.
We are building all the test suites (with `tests: true`) and cabal doesnt have `runnable: fase` 😉

Reply:
Yeah, fair.

Reply:
An alternative is mark them as ignored or broken for that hls version using tasty modifiers, but we are not using them that way for now